devices/HumanStateProvider/HumanStateProvider.cpp

```diff
@@ -435,6 +436,19 @@ bool HumanStateProvider::open(yarp::os::Searchable& config)
         }
     }
 
+     if (!(config.check("calibrationJointConfiguration") && config.find("calibrationJointConfiguration").isList())) {
+        yInfo() << LogPrefix << "calibrationJointConfiguration option not found or not valid, using zero configuration instead.";
+        pImpl->calibrationJointConfiguration.clear();
+    }
+    else
+    {
+        auto calibrationJointConfigurationBottle = config.find("calibrationJointConfiguration").asList();
+        for (size_t it = 0; it < calibrationJointConfigurationBottle->size(); it++)
+        {
```

Comment:
This **should** address https://github.com/robotology/human-dynamics-estimation/pull/341#discussion_r1128127974:

```suggestion
        auto calibrationJointConfigurationBottle = config.find("calibrationJointConfiguration").asList();
        if(calibrationJointConfigurationBottle->size()!=jointList.size())
        {
                yError() << LogPrefix << "Parameter calibrationJointConfiguration must have the same size of the list of selected joints!";
                return false;
        }
        for (size_t it = 0; it < calibrationJointConfigurationBottle->size(); it++)
        {
```

Reply:
`jointList.size()` was giving an identifier error. I solved by creating again the list from the parameter.

Reply:
I don't think it is very safe or efficient to do that. Probably you just needed to use ``pImpl->jointList`` instead of ``jointList``.

Reply:
It worked. Fixed here https://github.com/robotology/human-dynamics-estimation/pull/341/commits/a762b36fdc86cc187f814c82744c96ab51728f75

Reply:
Seems that CI workflow fails somewhere. In my machine it builds just fine so at this point I'm in your hand guys.

Reply:
Seems that these fails were a momentary problem server-side.